client/navigation/components/container/index.js

```diff
@@ -114,6 +114,8 @@ const Container = ( { menuItems } ) => {
 		} );
 	};
 
+	const isRootLevel = activeLevel === 'woocommerce' && rootBackUrl;
```

Comment:
Nitpick: this variable name may be slightly misleading (just worried someone else may try to reuse it without noticing the `rootBackUrl` at the end.

Maybe `isRootBackVisible` or something along those lines.

Reply:
Good call, fixed :+1: